core/pom.xml

```diff
@@ -65,13 +65,14 @@
     </scm>
     <properties>
         <jahia-module-type>system</jahia-module-type>
-        <jahia-depends>serverSettings</jahia-depends>
+        <jahia-depends>dx-commons-webpack,jahia-ui-root,serverSettings</jahia-depends>
```

Comment:
Is this really needed ? That does not looks like a hard requirement

Reply:
Well, the registration/menu item part is not going to work without those two modules, so I figured it could make sense to make it more explicit. But I see your point and I have no issue with not having it there.

Reply:
this will prevent the module to start if jahia-ui-root is not present (but the module can eventually be used without the navigation), and it will even force a full refresh of the module if jahia-ui-root is updated. This is particularly a problem for external-provider , which have a lot of dependant modules and will cascade the refresh .. but also valid for all other server settings